data/formats-data.js

```diff
@@ -6274,7 +6335,7 @@ exports.BattleFormatsData = {
 	},
 	pheromosa: {
 		randomBattleMoves: ["highjumpkick", "uturn", "icebeam", "poisonjab", "bugbuzz"],
-		randomDoubleBattleMoves: ["highjumpkick", "uturn", "icebeam", "poisonjab", "bugbuzz", "protect"],
+		randomDoubleBattleMoves: ["highjumpkick", "uturn", "icebeam", "poisonjab", "bugbuzz", "protect", "lowkick", "speedswap"],
```

Comment:
It's good for easing predictions against Protect on any other mon, but Pheromosa is such an easy target that if it hits something that's using Protect, it's probably dead either way. Removing.